app/vmalert/datasource/vm.go

```diff
@@ -70,7 +73,12 @@ func (s *VMStorage) Query(ctx context.Context, query string) ([]Metric, error) {
 	const (
 		statusSuccess, statusError, rtVector = "success", "error", "vector"
 	)
-	req, err := http.NewRequest("POST", s.queryURL+url.QueryEscape(query), nil)
+	q := s.queryURL + url.QueryEscape(query)
+	if s.lookBack > 0 {
+		lookBack := time.Now().UTC().Add(-s.lookBack)
```

Comment:
do we really need UTC here?

Reply:
Yes. As I understand, VM and Prom always use UTC for timestamps. Plz correct me if I'm wrong @valyala

Reply:
It looks like `UTC()` isn't needed here, since it just updates the timezone field inside `time.Time` struct without updating timestamp field. When the time is converted to unix timestamp with `time.Unix()`, then the timezone isn't taken into account.